src/collect-location.rst

```diff
@@ -7,19 +7,25 @@ ODK Collect uses Android's `Fused Location Provider <https://developers.google.c
 
 Android Location Settings
 --------------------------
 
-Different Android versions provide different settings for controlling how geolocation is captured. See how to configure Android location `in the Android documentation <https://support.google.com/accounts/answer/3467281>`_. In general, the more different sensors are available to Android, the better its geolocation estimation will be so make sure they are all enabled.
+Make sure that Android is configured to capture the highest-accuracy location possible. Different Android versions provide different settings for controlling how geolocation is captured. See how to configure Android location `in the Android documentation <https://support.google.com/accounts/answer/3467281>`_. In general, the more different sensors are available to Android, the better its geolocation estimation will be so make sure they are all enabled.
 
 Helping the GPS Sensor
 ------------------------
 
 GPS works by receiving signals from specific satellites so to help your device get a geolocation reading, get as clear of a view of the sky as possible. GPS does not work inside of buildings and will be affected by obstacles like dense forest canopy or tall buildings. It is typically the main technology used to identify a device's geolocation anywhere other than in urban areas where multiple WiFi access points and cell towers can be accessed at the same time. 
 
-It can take a while to get satellite lock when first attempting to get a geolocation. You can use a :ref:`background location question <metadata-start-geopoint>` at the beginning of your form definition to "warm" the GPS before data collectors to answer a :ref:`location question <location-widgets>`. You can also use an external app such as `GPS Status & Toolbox <https://play.google.com/store/apps/details?id=com.eclipsim.gpsstatus2>`_ for this purpose. Additionally, this application can help you troubleshoot GPS issues. See `its documentation <https://mobiwia.com/gpsstatus/>`_ for more details.
\ No newline at end of file
+It can take a while to get satellite lock when first attempting to get a geolocation. You can use a :ref:`background location question <metadata-start-geopoint>` at the beginning of your form definition to "warm" the GPS before data collectors to answer a :ref:`location question <location-widgets>`. You can also use an external app such as `GPS Status & Toolbox <https://play.google.com/store/apps/details?id=com.eclipsim.gpsstatus2>`_ for this purpose. Additionally, this application can help you troubleshoot GPS issues. See `its documentation <https://mobiwia.com/gpsstatus/>`_ for more details.
```

Comment:
The last two sentences about troubleshooting really stick out to me. Remove them or put them in a tip and maybe link to the FAQ (https://mobiwia.com/gpsstatus/faq/) which, to be honest, isn't that helpful for troubleshooting.